docs/rules/prefer-key-over-key-code.md

```diff
@@ -0,0 +1,33 @@
+# Enforce the use of event.key instead of event.keyCode
```

Comment:
```suggestion
# Prefer `event.key` over `event.keyCode`
```

Same with the description in the readme.

Reply:
Done